azkaban-common/src/main/java/azkaban/utils/AzkabanPatternLayout.java

```diff
@@ -0,0 +1,112 @@
+package azkaban.utils;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+
+import org.apache.log4j.helpers.PatternConverter;
+import org.apache.log4j.helpers.PatternParser;
+import org.apache.log4j.PatternLayout;
+import org.apache.log4j.spi.LoggingEvent;
+
+
+/**
```

Comment:
I didn't touch the original file. This file is newly created. using this new class needs conf change as well.